cmd/edit/edit.go

```diff
@@ -0,0 +1,52 @@
+package edit
+
+import (
+	"fmt"
+	"io/ioutil"
+	"encoding/json"
+
+	"github.com/dnote-io/cli/utils"
+)
+
+func Edit(note_nu string, newcontent string) error {
```

Comment:
@michielboekhoff note_nu, represents note_name/uid, I cant think of a better name when i wrote it